### PR TITLE
Fix to spatial color range.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/3862

This simplifies things quite a bit. I can't remember why I chose to do it the other way. Hopefully Travis will catch it if I broke something.

### CODE REVIEW

**Comment by jaeilepp** [comment]
Just realized that we probably shouldn't do inplace operations here.

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -146,16 +146,13 @@ def _topo_closed(events, ax, lines, fill):
...
-        dim_min = this_pos.min()
-        dim_max = (this_pos - dim_min).max()
-        dim -= dim_min
-        dim /= dim_max
-    return np.asarray([x, y, z]).T
+    r, g, b = x.copy(), y.copy(), z.copy()
+    for dim in [r, g, b]:
+        dim -= dim.min()
+        dim /= dim.max()
+    return np.asarray([r, g, b]).T
```
Comment (Eric89GXL) [inline mne/viz/evoked.py]: You can do this faster and in fewer lines with:
```
rgb = np.array([x, y, z]).T
rgb -= rgb.min(0)
rgb /= rgb.max(0)
```

**Comment by jona-sassenhagen** [comment]
Looks much better.

**Comment by agramfort** [comment]
good on your end @jona-sassenhagen ?

**Comment by jona-sassenhagen** [comment]
+1

**Comment by agramfort** [comment]
cool 

@jaeilepp merge if you're done

thx

**Pull request merged**